xt-prod-devel-rpi5-domd/recipes-extended/xen/files/xen-scmi.cfg

```diff
@@ -0,0 +1,2 @@
+CONFIG_ARM_SCI=y
```

Comment:
Is this change should be in this  commit? Commit message does not mention enabling SCMI